gdiObjects.xBar version 2.0.2112.20

gdiObjects opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/38368)

gdiObjects committed:
gdiObjects.xBar version 2.0.2112.20

ItzLevvie commented on manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml:
```diff
@@ -0,0 +1,30 @@
+Tags:
+- sidebar
+- appbar
+- windows 10
+- windows 11
```
```suggestion
- windows-10
- windows-11
```

Trenly replied:
I would argue that these tags should be removed entirely, as they are what I would consider to be "low-value" tags, in that they don't provide any meaningful information about the application itself or it's uses. They simply denote which versions of windows the application can be installed on, which could easily be covered by adding the MinimumOSVersion

ItzLevvie commented on manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.installer.yaml:
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: gdiObjects.xBar
+PackageVersion: 2.0.2112.20
+Installers:
+- Architecture: x86
```
```suggestion
- Architecture: x64
```

ItzLevvie commented on manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml:
```diff
@@ -0,0 +1,30 @@
+License: Copyright © 2008-2021 gdiObjects
+ShortDescription: A Dockable Infobar and App Launcher for Windows
+Description: The Fast & Easy Way to Find and Launch Software Apps, Open Documents, or Access Folders, and Drives in Windows. Quickly Organize Shortcuts into User-defined Categories. View System Details and Monitor your PC’s Performance with CPU, Memory, Temperature, and Battery Sensors.
+Moniker: xBar
```
```suggestion
Moniker: xbar
```
This is meant to be in lower-case.

gdiObjects committed:
Update manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

gdiObjects committed:
Update manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.installer.yaml

The app is 64-bit, but the installer is 32-bit.  Does this field represent the App or Installer?

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>

gdiObjects committed:
Update manifests/g/gdiObjects/xBar/2.0.2112.20/gdiObjects.xBar.locale.en-US.yaml

Co-authored-by: Levvie - she/her <11600822+ItzLevvie@users.noreply.github.com>
